packages/governance/test/unitTests/test-ballotCount.js

```diff
@@ -1,11 +1,11 @@
 // @ts-check
 
-import { test } from '@agoric/zoe/tools/prepare-test-env-ava';
-import '@agoric/zoe/exported';
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import '@agoric/zoe/exported.js';
 import { E } from '@agoric/eventual-send';
```

Comment:
What's the difference between imports that need a `.js` extension and ones that don't?  Is it possible to consistently always include the extension, or do we have to know something about the format of the source to decide how to punctuate the import statement?